monitoring/collect_and_push_job_stats.py

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env python
```

Comment:
Shouldn't it be `python3`? Also, do you think it's a good idea to `chmod +x` these files?